pkg/plugins/policies/core/rules/rules.go

```diff
@@ -73,10 +76,53 @@ func (ss Subset) IsSubset(other Subset) bool {
 			return false
 		}
 		for _, otherTag := range oTags {
-			if otherTag.Value == tag.Value && otherTag.Not != tag.Not {
+			if !singleDimSubset(tag, otherTag) {
```

Comment:
nit but IMO would be easier to understand without the negation

Reply:
I renamed the method to `isSubset` so I think now it's understandable with negation